devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -383,4 +534,20 @@ private logDebug(msg) {
 
 private logTrace(msg) {
     log.trace "$msg"
+}
+
+private isButtonAvailable() {
+    if(device == null){
```

Comment:
nit: Please add a space after `if` and before `{`

Reply:
OK! I modified it.